src/types/utils/deferred-promise.ts

```diff
@@ -0,0 +1,6 @@
+export type DeferredPromise = {
```

Comment:
We have DeferredPromise in playkit-js as well, can we use type from there?

Reply:
Right , moved it to there and forgot to remove it here